### PR TITLE
Catch multiple exceptions

### PR DESCRIPTION
**Pull request by evgeniyamelone**
### Exceptions Practice #4: Catch Multiple Exceptions

- Create three new types of exceptions. 
- Write a class with a method that throws all three. 
- In main(), call the method but only use a single catch clause that will catch all three types of exceptions.

### CODE REVIEW

File: .idea/Java_Edu.iml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
```
Comment (mih-kopylov) [inline .idea/Java_Edu.iml:2]: Same thing here: would be nice not to have not related files in PR

File: src/main/java/catchMultipleExeptions/Demo.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+public class Demo {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter numerator:");
+        int numerator = scanner.nextInt();
+        System.out.println("Enter denominator:");
+        int denominator = scanner.nextInt();
+
+        try {
+            new Arithmetic();
```
Comment (mih-kopylov) [inline src/main/java/catchMultipleExeptions/Demo.java]: Same thing here: we don't need to create an instance of a class to call a static method

File: src/main/java/catchMultipleExeptions/ResultZeroException.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package catchMultipleExeptions;
+
+public class ResultZeroException extends Exception {
+
+    public ResultZeroException(String exceptionString) {
```
Comment (mih-kopylov) [inline src/main/java/catchMultipleExeptions/ResultZeroException.java]: nit: parameter is usually called `message`

**Review by mih-kopylov — CHANGES_REQUESTED** [review]

File: src/main/java/catchMultipleExeptions/Arithmetic.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        try {
+            if (denominator == 0)
+                throw new InvalidInputParamException("Denominator can't be zero");
+            else if (denominator > numerator)
+                throw new ResultZeroException("Denominator can't be greater than numerator");
+            else if (denominator == numerator)
+                throw new EqualNumbersException("Denominator can't be equal to numerator");
+            else
+                return numerator/denominator;
+        } catch (Exception e) {
```
Comment (ssheludchenko) [inline src/main/java/catchMultipleExeptions/Arithmetic.java]: This is just pointless to throw and exception and try to catch it immediately, especially if no more value added to that.
You can remove `try ... catch` block here.

**Review by ssheludchenko — CHANGES_REQUESTED** [review]

File: src/main/java/catchMultipleExeptions/Arithmetic.java (hunk 4 above)
Comment (evgeniyamelone) [reply src/main/java/catchMultipleExeptions/Arithmetic.java]: removed

**Pull request merged**